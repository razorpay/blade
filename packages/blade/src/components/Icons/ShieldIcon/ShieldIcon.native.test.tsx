import ShieldIcon from './';
import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';

describe('<ShieldIcon />', () => {
  it('should render ShieldIcon', () => {
    const renderTree = renderWithTheme(
      <ShieldIcon color="feedback.icon.neutral.lowContrast" size="large" />,
    ).toJSON();
    expect(renderTree).toMatchSnapshot();
  });
});
