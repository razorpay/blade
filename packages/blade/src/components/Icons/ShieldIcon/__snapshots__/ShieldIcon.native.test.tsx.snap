// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShieldIcon /> should render ShieldIcon 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <RNSVGSvgView
    accessibilityElementsHidden={true}
    align="xMidYMid"
    bbHeight="20px"
    bbWidth="20px"
    data-blade-component="icon"
    fill="none"
    focusable={false}
    height="20px"
    importantForAccessibility="no-hide-descendants"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        [
          {},
        ],
        {
          "flex": 0,
          "height": 20,
          "width": 20,
        },
      ]
    }
    vbHeight={24}
    vbWidth={24}
    width="20px"
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M11.7575 1.02986C11.9167 0.990047 12.0833 0.990047 12.2425 1.02986L20.2425 3.02986C20.6877 3.14115 21 3.54113 21 4V12C21 15.4464 18.7183 18.2003 16.6585 20.0026C15.6076 20.9221 14.5615 21.6408 13.78 22.1292C13.3882 22.3741 13.0603 22.5627 12.8281 22.6913C12.7119 22.7556 12.6194 22.805 12.5547 22.839C12.5223 22.856 12.4968 22.8692 12.4788 22.8784L12.4573 22.8894L12.4508 22.8926L12.4486 22.8937C12.4486 22.8937 12.4472 22.8944 12 22C11.5528 22.8944 11.5522 22.8941 11.5522 22.8941L11.5492 22.8926L11.5427 22.8894L11.5212 22.8784C11.5032 22.8692 11.4777 22.856 11.4453 22.839C11.3806 22.805 11.2881 22.7556 11.1719 22.6913C10.9397 22.5627 10.6118 22.3741 10.22 22.1292C9.43854 21.6408 8.39238 20.9221 7.3415 20.0026C5.28175 18.2003 3 15.4464 3 12V4C3 3.54113 3.3123 3.14115 3.75746 3.02986L11.7575 1.02986ZM12 22L11.5522 22.8941C11.8337 23.0349 12.1657 23.0352 12.4472 22.8944L12 22ZM12 20.8628C12.1916 20.7541 12.4367 20.6103 12.72 20.4333C13.4385 19.9842 14.3924 19.3279 15.3415 18.4974C17.2817 16.7997 19 14.5536 19 12V4.78078L12 3.03078L5 4.78078V12C5 14.5536 6.71825 16.7997 8.6585 18.4974C9.60762 19.3279 10.5615 19.9842 11.28 20.4333C11.5633 20.6103 11.8084 20.7541 12 20.8628Z"
        fill={
          {
            "payload": 4280563015,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
